packages/mobile/src/identity/commentEncryption.test.ts

```diff
@@ -0,0 +1,100 @@
+import {
```

Comment:
Actually a rename of `identity/commentKey.ts` but I'll be significantly increasing the coverage we had there before merging this.